### DISCUSSION TITLE
Just a thanks!

### DISCUSSION THREAD

**Discussion by boldandbusted** [sorayuki/obs-multi-rtmp#454] [General]
I use OBS on Linux (version 30.2.3 currently), and have successfully used this plugin (version "6.0.1") with YouTube RTMP, and LinkedIn RTMP. After figuring out how to properly show the Dock in OBS that is generated by this plugin, it has worked well. This plugin fills a gap that isn't yet met by the OBS core nor by other plugins.

So, I want to say THANK YOU to the devs and community here. 我很感激你的工作! I hope to donate at some point in the future. Cheers! 祝你长寿！

**Comment by frdbr** [comment]
Could you please share how you managed to use it under linux?
